### PR TITLE
Completed the JavaScript Forms exercise

### PR DESCRIPTION
**Pull request by owen-adams2021**
image

### CODE REVIEW

File: javascript-forms/form-events.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+var $name = document.forms.myform.elements.name;
+var $email = document.querySelector('#user-email');
+var $message = document.forms[0].elements[2];
```
Comment (thebearingedge) [inline javascript-forms/form-events.js:3]: 🤔 Try grabbing each of these the same way.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
One minor tweak.

File: javascript-forms/form-events.js (hunk 1 above)
Comment (owen-adams2021) [reply javascript-forms/form-events.js:3]: I just did it for exercise :) Will do!

**Comment by owen-adams2021** [comment]
image

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**